recipe/meta.yaml

```diff
@@ -10,18 +10,20 @@ source:
   sha256: 46f0c74d73efa723776764b43c9739f68052495dd3d734319c1d0eb58511f15b
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  skipt: true  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
-    - setuptools-scm
+    - setuptools-scm >=5
+    - setuptools >=46.1.0
```

Comment:
`toml` is missing https://github.com/pyscaffold/configupdater/blob/0539c207ac57ea4fa6a64618e41b987c5865ffd9/pyproject.toml#L3

Reply:
Added, thanks!